app/models/suggestion.py

```diff
@@ -0,0 +1,119 @@
+from .. import db
+from datetime import datetime
+import pytz
+
+
+class Suggestion(db.Model):
+    """
+    Association between a resource and potential suggestions for it
+    """
+    __tablename__ = 'suggestions'
+    id = db.Column(db.Integer, primary_key=True)
+    # resource_id is -1 for insertion
```

Comment:
We don't need this comment anymore. If there is no resource, then the relationship will be `null`.